src/science/soilsnow/cbl_soilsnow_main.F90

```diff
@@ -83,16 +84,15 @@ SUBROUTINE soil_snow(dels, soil, ssnow, canopy, met, bal, veg)
     ssnow%dtmlt = 0.0
     ssnow%osnowd = ssnow%snowd
 
-
-    wbliq = ssnow%wb - ssnow%wbice
+    ssnow%wbliq = ssnow%wb - ssnow%wbice
```

Comment:
this is a bug in main - it being local means it stays constant thru the run

Reply:
There is not much I can say here of value except that the GW work did not alter in any way cbl_soilsnow_main.F90 (I double checked). I am not sure however if breaking the old soil_snow single module into many here introduced bugs, but that's of course unrelated to the GW work.

Reply:
that refactoring  was done long ago my friend. Thanks for looking at it